plant/plants/api/serializers.py

```diff
@@ -44,18 +55,20 @@ class Meta:
             "owner",
             "staff",
             "staff_ids",
+            "family_id",
+            "family",
         ]
 
     def create(self, validated_data):
-        staff = validated_data.pop("staff_ids")
         instance = super().create(validated_data)
-        instance.staff.set(staff)
+        validated_data["family"] = validated_data.pop("family_id")
+        validated_data["family"] = validated_data.pop("staff_ids")
```

Comment:
Your previous assigning of staff was correct as side many of relation can't be done directly. So please restore
```
staff = validated_data.pop("staff_ids")
instance.staff.set(staff)
```
and remove L65